src/doc/ESMF_debug.tex

```diff
@@ -155,3 +155,24 @@
 very top of the stack {\tt esmApp.F90}, and at the very bottom {\tt ocn.F90}.
 In fact the location of the division by zero is correctly identified by the
 runtime library at line 149 in file {\tt ocn.F90}.
+
+It is possible to force a hard crash within the ESMF library while logging
+ERRORS or WARNINGS to the {\tt PET<nnn>.ESMF\_LogFile}. Doing so can be
+advantageous because it may produces a code dump and/or backtrace at the
+initial point of error without ESMF return code handling. There are two
+settings controlling log message error handling. The first
+{\tt ESMF\_RUNTIME\_LOGMSGABORT} configures the ESMF library to abort during
+specified log message types, such as {\tt ESMF\_LOGMSG\_ERROR}. The second
```

Comment:
It would help to provide an example of how this can be set. My understanding from looking briefly at the code is that this can be set to a single log message type or a list of multiple types. That wasn't totally clear to me from reading through the documentation and I think a couple of examples would help clarify this.